packages/components/select/src/select.vue

```diff
@@ -327,10 +327,7 @@ export default defineComponent({
       type: [String, Object] as PropType<string | Component>,
       default: ArrowUp,
     },
-    tagType: {
-      type: String,
-      default: 'info',
-    },
+    tagType: tagProps['type'],
```

Comment:
Use `tagProps.type` instead of `tagProps['type']`

Reply:
Done.